src/main/java/hibernate/tournament/model/MatchDto.java

```diff
@@ -0,0 +1,39 @@
+package hibernate.tournament.model;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "results")
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@AllArgsConstructor
```

Comment:
You have IDENTITY generator for id.
It does not make sense to use `lombok.AllArgsConstructor`